bindings/rust/wasmedge-sys/src/utils.rs

```diff
@@ -62,3 +69,249 @@ pub fn log_debug_info() {
 pub fn log_error_info() {
     unsafe { ffi::WasmEdge_LogSetErrorLevel() }
 }
+
+/// Converts WasmEdge_Result to WasmEdgeResult
+pub(crate) fn check(result: WasmEdge_Result) -> WasmEdgeResult<()> {
```

Comment:
`check` is very confusing.

Reply:
Accepted. Will refine the doc comment. Thanks!

Reply:
I am not sure `check` is meaningful. This function seems like that trying to convert `WasmEdge_Result` to `WasmEdgeResult`.

Reply:
From the perspective of the internal implementation of this function, yes, you're right, it actually converts the raw `WasmEdge_Result` into the result type of `wasmedge-sys`. But, consider the following use of the function in the practice. The semantics of `check` could be better for a user to understand the code, while `convert` could result in some kind of "misleading". Hence, in this view, using "check" should be better. Thanks!

```rust
let path = utils::path_to_cstring(path.as_ref())?;
let mod_name: WasmEdgeString = mod_name.as_ref().into();
unsafe {
    check(ffi::WasmEdge_VMRegisterModuleFromFile(self.inner.0, mod_name.as_raw(), path.as_ptr()))?
};

Ok(())
```